examples/test-app/qbec.yaml

```diff
@@ -25,6 +25,10 @@ spec:
       - name: externalFoo
         default: 'bar'
   environments:
+    local:
+      context: minikube
+      properties:
+        envType: local
```

Comment:
How is envType used?

Reply:
This is a bag of aribtrary properties. qbec doesn't know or care about stuff under `properties`

Reply:
it can be accessed in jsonnet as `std.extVar('qbec.io/envProperties')`